pkg/audit/manager.go

```diff
@@ -757,6 +771,19 @@ func (am *Manager) addAuditResponsesToUpdateLists(
 		if *emitAuditEvents {
 			emitEvent(r.Constraint, timestamp, ea, gvk, namespace, name, r.Msg, am.gkNamespace, am.eventRecorder)
 		}
+
+		if *logEvalMetricsAudit {
+			am.log.WithValues(
+				logging.Process, "audit",
+				logging.EventType, "violation",
+				logging.ConstraintName, r.Constraint.GetName(),
+				logging.ConstraintGroup, r.Constraint.GroupVersionKind().Group,
+				logging.ConstraintAPIVersion, r.Constraint.GroupVersionKind().Version,
+				logging.ConstraintKind, r.Constraint.GetKind(),
+				logging.ConstraintAction, r.EnforcementAction,
+				logging.EvaluationMetrics, r.EvaluationMeta,
```

Comment:
Do we know that EvaluationMeta will always contain evaluation metrics? I don't think that's necessarily implied by the name.